src/lib.rs

```diff
@@ -190,6 +190,7 @@ mod_use!(cfg(target_os = "redox"), use_file);
 mod_use!(cfg(target_os = "solaris"), solaris_illumos);
 mod_use!(cfg(windows), windows);
 mod_use!(cfg(target_env = "sgx"), rdrand);
+mod_use!(cfg(target_os = "uefi"), rdrand);
```

Comment:
I think we should use `all(target_os = "uefi", target_arch="x86_64")`.

Reply:
Fixed here and below